daml/Marketplace/Clearing/Role.daml

```diff
@@ -111,3 +111,20 @@ template Request
       Reject : ()
         do
           return ()
+
+template RequestRoleTermination
```

Comment:
Is using request contracts a design decision to require confirmation by the operator to terminate a role? If not, why not just add a consuming `Terminate` choice on the role contract with the `provider` as the `controller`? That should work, unless I'm missing something

Reply:
The Roles contracts have operator and provider as signatories

Reply:
Right - but having a choice on the contract with only the `provider` as the controller would give the controller party authorization to archive the contract (provided it is within that choice), without explicit permission during the exercise from the other signatory

Reply:
I'll give it a try - I like the idea of having a terminate choice better than the request templates